src/games/strategy/performance/EnablePerformanceLoggingCheckBox.java

```diff
@@ -10,12 +10,12 @@
 
   public EnablePerformanceLoggingCheckBox() {
     super("Enable Performance Logging");
-    setSelected(Perf.isEnabled());
+    setSelected(PerfTimer.isEnabled());
```

Comment:
Are you sure about this update here? I know one weakness of the timer classes was Perf probably should have been called PerfTimer, but regardless, Perf is the only thing that should be touching PerfTimer. You get a PerfTimer from Perf, and then the auto-close try/catch blocks closes the PerfTimer. In fact we probably should make all methods in PerfTimer private/protected except for close() to enforce that relationship.